spec/requests/vendor_api/make_an_offer_spec.rb

```diff
@@ -0,0 +1,71 @@
+require 'rails_helper'
+
+RSpec.describe 'Vendor API - POST /api/v1/applications/:id/offer', type: :request do
```

Comment:
What would you think about making this consistent with the [existing request specs like those for the /applications endpoint](https://github.com/DFE-Digital/apply-for-postgraduate-teacher-training/blob/de4160ea9e324f76762c38599a6a261f4b8eaf90/spec/requests/vendor_api/get_multiple_applications_spec.rb)? In those we're writing the tests in flat `it` blocks with multiple assertions, without `context`, `before` and `let`. 

The multiple assertions speeds up the tests, because we don't have to make the requests for every `it`.

Regarding `let`, I'm a big fan of the ThoughtBot approach to those:

https://thoughtbot.com/blog/lets-not
https://thoughtbot.com/blog/my-issues-with-let
https://thoughtbot.com/blog/four-phase-test

Reply:
@tijmenb I quite like the `context -> let` idiom but I agree it's overkill here. Will condense